src/controllers/DynamicFacetQueryController.ts

```diff
@@ -13,7 +13,7 @@ export class DynamicFacetQueryController {
   private numberOfValuesToRequest: number;
   private freezeCurrentValues = false;
   private freezeFacetOrder = false;
-  protected preventAutoSelection = false;
+  private preventAutoSelection = false;
```

Comment:
Any chance this could be breaking to someone?

Reply:
Extremely unlikely. This is the DynamicFacetQueryController which isn't exported. We are extending this class ourselves to use that value, but the autoselection is not available for ranges